airflow/contrib/operators/dataproc_operator.py

```diff
@@ -688,6 +688,10 @@ class DataProcPigOperator(BaseOperator):
     :type delegate_to: str
     :param region: The specified region where the dataproc cluster is created.
     :type region: str
+    :param job_error_states: Job states that should be considered error states.
+        Any states in this list will result in an error being raised and failure of the
+        task. Defaults to ``['ERROR']``.
```

Comment:
Can you add an example here? For eg. something along the lines of "If you want to consider "CANCELLED" as errored state add ["CANCELLED"]". Also add all the possible states with "Possible Values: "

Reply:
Just saw these... Will do.

Reply:
Just pushed...